src/test/java/ch/astorm/jotlmsg/OutlookMessageMSGTest.java

```diff
@@ -68,6 +71,11 @@ public void testMessageWithAttachment2() throws Exception {
         message.addAttachment("message3.txt", "text/html", new ByteArrayInputStream("<html><body>Some html page</body></html>".getBytes(StandardCharsets.UTF_8)));
         
         testMessage(message);
+        
+//        // There was a bug regarding the mimetype, so just generate a new reference file.
+//        File msgFile = new File("src/test/resources/ch/astorm/jotlmsg/generated/with-attachments-2.msg");        
+//        message.writeTo(msgFile);
```

Comment:
Since this has been fixed, could you please rebase and regenerate it ?

Reply:
Yes, sure.